vendor/dataTables.datetime-moment.js

```diff
@@ -47,7 +47,7 @@
         return 'moment-' + format;
       }
 
-      return moment( d, format, locale, true ).isValid() ?
```

Comment:
1. please explain all changes to this file
2. making changes to a vendor library seems unnecessary and it means upgrading this dependency is more difficult, is this the only option we have?

Reply:
@tonytlwu see if you have anything to add here

Reply:
Not much more to add. I agree with the above. More explanation on the assessment and proposed solutions please as well as any other tried and failed solutions.

Reply:
@squallstar 1. We simply disabled strict mode when we parsing a date. With the strict mode on, we have to get an error all the time. Could be because of the format that we use.
2. That is not exactly a library. That is a custom plugin that was added by us to control the sorting within the `DataTable` library in this [PR](https://github.com/Fliplet/fliplet-widget-organization-dashboard/pull/21). And this is the only place where we determine how we should sort the date.

Reply:
@YaroslavOvdii I am still confused at why you needed to make a change to `dataTables.datetime-moment.js`. Surely the source (input) date can be formatted in the format the library expects?